### PR TITLE
[Windows] Make `yarn pkg-version` work across platforms

### PR DESCRIPTION
**Pull request by AMoo-Miki**
Signed-off-by: Miki <amoo_miki@yahoo.com>

### Description
`yarn pkg-version` is used by CI to fetch the current version of Dashboards. This commit makes it work across platforms.
 
### Check List
- [X] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [X] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by ananzh — DISMISSED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
     "docs:acceptApiChanges": "node --max-old-space-size=6144 scripts/check_published_api_changes.js --accept",
     "osd:bootstrap": "node scripts/build_ts_refs && node scripts/register_git_hook",
     "spec_to_console": "node scripts/spec_to_console",
-    "pkg-version": "./dev-tools/get-version.sh"
+    "pkg-version": "node -e \"console.log(require('./package.json').version)\""
```
Comment (tmarkley) [inline package.json:74]: This is a tiny bit scary but I think it's pretty straightforward and safe for the CI workflows. Can we delete `get-version.sh` too?

Comment (joshuarrrr) [reply package.json:74]: +1 - we should remove `get-version.sh` if we're no longer using.

Comment (AMoo-Miki) [reply package.json:74]: It is done.

**Review by joshuarrrr — APPROVED** [review]

**Review by tmarkley — APPROVED** [review]
Looks good!

**Review by ananzh — APPROVED** [review]

**Comment by AMoo-Miki** [comment]
backport 2.x went through.

**Pull request merged**